src/finch/util/config.py

```diff
@@ -0,0 +1,70 @@
+import os
+import shutil
+import sys
+import sysconfig
```

Comment:
[`sysconfig`](https://docs.python.org/3/library/sysconfig.html) is meant for querying how Python was built, which is usually very different from the system it's run on.

Reply:
I see. I was hoping that `sysconfig.get_path("data")` would allow me to have different Finch caches in each virtualenv.

Reply:
That's fine, my comment was more about `CC` and `CXX`.

Reply:
I don't know of a standard way.

Reply:
for now, I've removed the calls to sysconfig for shared_library_suffix and cc, in favor of more generic defaults. We'll fix it in a separate PR, as you mentioned.